flux/Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT_GOAL := get-upstream
+
+
+# https://github.com/fluxcd/flux2
+VERSION=v0.32.0
+
+# tool 'yq' can be downloaded from https://github.com/mikefarah/yq
+
+
+# This will print type of resources found in upstream but only required once are added
+.PHONY: get-upstream-resource-types
+get-upstream-resource-types:
+	curl -Ls https://github.com/fluxcd/flux2/releases/download/$(VERSION)/install.yaml > /tmp/flux.yaml
+	cat /tmp/flux.yaml | rg ^kind: | uniq
+	rm  /tmp/flux.yaml
+
+# This fetches manifests from upstream and splits resources into cluster/ and
+# namespaced/ directories based on type of resources. 
+.PHONY: get-upstream
+get-upstream:
+	curl -Ls https://github.com/fluxcd/flux2/releases/download/$(VERSION)/install.yaml > /tmp/flux.yaml
+	
+	# replace flux-system with sys-flux (extra "" is needed in MacOS)
+	sed -i "" 's/flux-system/sys-flux/g' /tmp/flux.yaml
```

Comment:
This is fine, but check out https://github.com/chmln/sd - we use that in few scripts as well

Reply:
done